include/librealsense2/rsutil.h

```diff
@@ -194,64 +35,48 @@ static void next_pixel_in_line(float curr[2], const float start[2], const float
     }
 }
 
-static bool is_pixel_in_line(const float curr[2], const float start[2], const float end[2])
+/* Helper inner function (not part of the API) */
```

Comment:
Then why is it here in a public header? Same for `next_pixel_in_line` and `adjust_2D_point_to_boundary`

Reply:
BTW, I see we expose these to Python for some reason... but I see no usage in examples or anywhere. Why do we have these?

Reply:
1. Refactored into cpp.
2. They shouldn't be exposed, can you remove it?